units/others/Rancher.cfg

```diff
@@ -0,0 +1,117 @@
+#textdomain wesnoth-A_Story_of_the_Northlands
+[unit_type]
+    id=Rancher
+    name= _ "Rancher"
+    race=human
+    image="units/humans/rancher.png"
+    profile="portraits/boy.png"
+    hitpoints=22
+    movement_type=mounted
+    movement=7
+    experience=28
+    level=0
+    alignment=lawful
+    advances_to=null
+    undead_variation=mounted
+    cost=12
+    usage=scout
+    description= _ "In the wilder, less tamed regions of Wesnoth, the horse clans provide a steady supply of mounts, so even common farmers can usually afford a horse."
```

Comment:
```suggestion
    description= _ "In the wilder, less tamed regions of Wesnoth, the horse clans provide a steady supply of mounts available to even common farmers."
```

Reply:
I'll want to update the textdomain for this file now that we've changed this string...